### DISCUSSION TITLE
Setup parameters

### DISCUSSION THREAD

**Discussion by stellaris-lunae** [GameServerManagers/LinuxGSM#4281] [Q&A] [answered]
I am wondering how do I change the commandline parameters?
Please no "just use start parameters", I have already tried that and I need to change the RCON password.

**Comment by dgibbs64** [comment]
All the info is available in the docs 😊 https://docs.linuxgsm.com/configuration/start-parameters

**Reply by stellaris-lunae** [reply]
Again, please no "just use startup parameters" as I already tried that

**Reply by dgibbs64** [reply]
Again, read the documentation. Thousands of other people manage to do this by reading the docs. You didn't even tell us what game you are trying to set up out of the 130+ available servers, provided no info, and expected an answer, while also being rude.

**Reply by stellaris-lunae** [reply]
i read the docs, they dont help, sorry, this is just a security risk i need to take care of, the game is rust too

**Reply by dgibbs64** [reply]
Update the setting in common.cfg. Make sure you have read and understand how LinuxGSM configs work https://docs.linuxgsm.com/configuration/linuxgsm-config
```
rconpassword="CHANGE_ME"
```

**Reply by stellaris-lunae** [reply, marked as answer]
I found my solution by referencing _default.cfg as a baseline.
